src/command.js

```diff
@@ -26,7 +26,15 @@ const Command = class {
   }
 
   static create (rawInputs) {
-    return doNotAllowMissingProperties(new this(rawInputs))
+    const command = doNotAllowMissingProperties(new this(rawInputs))
+    this._validateCreation(command)
+    return command
+  }
+
+  static _validateCreation (newlyCreatedCommandInstance) {
+    if (isNull(newlyCreatedCommandInstance.schema) || isUndefined(newlyCreatedCommandInstance.schema)) {
+      newlyCreatedCommandInstance.constructor.schema = {}
```

Comment:
In preparation for the static schema only.

Reply:
Interesting, are static fields not inherited or something?

I'm wondering if it's enough to just do `static schema = {}` here

It makes me nervous to mutate the class of the command when it's created.

Another option I can think of here is to do

`static create(inputs = {})`

which might solve the problem as well, though you'd still get undefined if explicitly asking for the schema.